src/reference/changes.rst

```diff
@@ -38,6 +38,14 @@ Cylc 8.3.0
    :cylc-uiserver: `1.4 <https://github.com/cylc/cylc-uiserver/blob/1.4.x/CHANGES.md>`__
    :cylc-rose: `1.4 <https://github.com/cylc/cylc-rose/blob/1.4.x/CHANGES.md>`__
 
+Cylc ignores `PYTHONPATH`
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Cylc now ignores `PYTHONPATH` to make it more robust to task
+environments which set this value. If you want to add to the Cylc
+environment itself, e.g. to install a Cylc extension,
+use `CYLC_PYTHONPATH`.
+
```

Comment:
I think this could do with elaboration (maybe link to another section)

Reply:
Also
```suggestion
Cylc now ignores ``PYTHONPATH`` to make it more robust to task
environments which set this value. If you want to add to the Cylc
environment itself, e.g. to install a Cylc extension,
use ``CYLC_PYTHONPATH``.

```

Reply:
> I think this could do with elaboration (maybe link to another section)

Agreed

Reply:
So is this only relevant to people _developing_ their own Cylc plugins?

Reply:
Yes.

Reply:
Or *installing* them.

Reply:
It's not clear to me what
> If you want to add to the Cylc environment itself, e.g. to install a Cylc extension, use ``CYLC_PYTHONPATH``.

means in practice. But if you're not too fussed about it, then neither am I

Reply:
https://github.com/cylc/cylc-doc/blob/bda41e6f11fdb0f5994499c4b393dd0992be7918/src/plugins/install/index.rst#L59-L60

Reply:
If this affects custom xtriggers and event handlers, wouldn't it be worth mentioning those?

Reply:
Yes please.

Reply:
- xtriggers : covered by existing change to xtriggers docs.
- event handlers: I've added a note about this

General: I've added these cases to the examples in changes.rst.